packages/react/src/grid/PageGrid.tsx

```diff
@@ -0,0 +1,16 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright (c) 2023 Gemeente Amsterdam
+ */
+
+import { ReactNode } from 'react'
+
+export type PageGridColumnNumber = 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9 | 10 | 11 | 12
```

Comment:
Find a way to have just four options for the grid on narrow screens.

Reply:
It's just two different types, right? `PageGridColumnNumberWide` and `PageGridColumnNumberNarrow`?

Reply:
Could be, but how would the component know on render when type to use or what the restriction is?

Reply:
I think the usual approach is to specify grid config per breakpoint, e.g.

```
export type GridCellProps = PropsWithChildren<{
  gridColumns?: { 
    narrow: [PageGridColumnNumberNarrow, PageGridColumnNumberNarrow] 
    wide: [PageGridColumnNumberWide, PageGridColumnNumberWide] 
  }
}>
```

To allow merging this PR today, I created a separate story in Jira for the narrow screens API.